cypress/support/fragments/inventory/inventoryInstance.js

```diff
@@ -254,6 +251,11 @@ export default {
       moveItemsButton.click()
     ]);
   },
+
+  closeItemView() {
```

Comment:
We have the same function in 'support\fragments\inventory\inventoryItem\itemView.js'

Reply:
closeDetailView();

Reply:
thx, fixed